docs/metrics.md

```
@@ -0,0 +1,125 @@
+
+## The bot: `metrics/cim-pageviews/CIMviews.py`
+
+A pywikibot script with two modes selected via `--mode`:
+
+### `--mode data` (default)
+
+For every Commons category that transcludes `{{views from category}}`:
+
+1. Walks `Category:Category with page views table` to enumerate target categories (removing the maintenance subcategory `Category:Category page views need to be updated`).
+2. Calls the Wikimedia Commons-Impact-Metrics REST endpoint:
+   ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)